wormhole-connect/src/config/utils.ts

```diff
@@ -43,6 +43,46 @@ export const validateChainResources = () => {
   validateResourceMap('rest');
 };
 
+export const mergeCustomTokensConfig = (
+  builtin: TokensConfig,
+  custom?: TokensConfig,
+): TokensConfig => {
+  if (!custom) return builtin;
+
+  const builtinTokens = Object.values(builtin);
+  const builtinSymbols = builtinTokens.map((tk) => tk.symbol);
+  const builtinKeys = builtinTokens.map((tk) => tk.key);
+
+  for (let key in custom) {
+    // Verify that custom token config does not conflict with any built-in tokens
+    const customToken = custom[key];
+    if (key in builtin) {
+      console.warn(
+        `Skipping custom token config for "${key}" because it conflicts with a built-in`,
+      );
+      continue;
+    }
+    if (builtinSymbols.includes(customToken.symbol)) {
+      console.warn(
+        `Skipping custom token config for "${key}" because its symbol "${customToken.symbol}" conflicts with a built-in`,
+      );
+      continue;
+    }
+    if (builtinKeys.includes(customToken.key)) {
+      console.warn(
+        `Skipping custom token config for "${key}" because its key "${customToken.key}" conflicts with a built-in`,
+      );
+      continue;
+    }
+
+    // Accept custom token config
+    console.warn(`Accepted custom token config for "${key}"`);
```

Comment:
seems odd a warning log level for a success process

Reply:
good point lol